target/maven-status/maven-compiler-plugin/testCompile/default-testCompile/createdFiles.lst

```diff
@@ -0,0 +1,10 @@
+hw1\DivTest.class
+hw1\SqrtTest.class
+hw2\ex1\HomePageTest.class
+hw2\BaseClassTest.class
+hw1\AbstractCalculatorTest.class
+hw2\ex2\DifferentElementsTest.class
+hw1\SumTest.class
+hw1\DataProvidersTest.class
+hw1\SubTest.class
+hw1\MultiplyTest.class
```

Comment:
`target` folder should be removed from the repository

Reply:
done